### PR TITLE
Tags Added.

### PR DESCRIPTION
**Pull request by atharvagadkari05**
In this PR I have made several changes:
1. Added Various tags
2. Tested tags (Manually).
3. Created File JS file for automated testing.

AUTOMATED TESTING IS NOT  WORKING. PLS REVIEW.

### CODE REVIEW

File: src/lib/eccentric.css [hunk 1]
```diff
@@ -126,4 +126,18 @@ h5, .et-h5 {
...
+  font-family: var(--et-sans-serif);
+  margin: 0;
+  color: var(--et-black);
+}
+
+.et-bold, b {
+  font-weight: bold;
+}
+
+.et-paragragh, .et-regular, p {
```
Comment (yashsehgal) [inline src/lib/eccentric.css:140]: There's no need to specify CSS at this block as the properties are already defined for this.
Also, We need to create a method named as `.et-semibold` and set font-weight as 600.

But that's fine, LGTM!

I will apply the remaining changes.

File: src/lib/eccentric.scss [hunk 2]
```diff
@@ -149,4 +149,18 @@ h5, .et-h5{
...
+font-family: var(--et-sans-serif);
+margin: 0;
+color: var(--et-black);
+}
+ 
+.et-bold,b{
+  font-weight: bold;
+}
+// To remove extra (unwanted) margin.
+.et-paragragh,.et-regular,p{
```
Comment (yashsehgal) [inline src/lib/eccentric.scss:164]: Same review as `eccentric.css`

**Review by yashsehgal — APPROVED** [review]
There are some changes required, which I will apply, so don't worry @atharvagadkari05 @Jaspreet099. Also, I will explain to you guys today that how you need to generate tests using JS. So be there tonight. 

Thanks for your contribution. I am approving this PR.

**Comment by yashsehgal** [comment]
Also, Atharva. You have made a PR from v2.0 to master. It's a merge. There's no need to do this right now. Although, I have seen the code and it's able to merge but be conscious next time.

**Comment by atharvagadkari05** [comment]
Thanks, I will take care of it.

**Pull request merged**